src/Build/FileAccess/ProcessData.cs

```diff
@@ -21,5 +23,11 @@ namespace Microsoft.Build.Framework.FileAccess
         uint ParentProcessId,
         DateTime CreationDateTime,
         DateTime ExitDateTime,
-        uint ExitCode);
+        uint ExitCode) : ITranslatable
+    {
+        void ITranslatable.Translate(ITranslator translator)
+        {
+            throw new NotImplementedException();
```

Comment:
This right?

Reply:
Ah, maybe that's one reason this is still a draft =P

Reply:
It sure wasn't :)